src/app/index.js

```diff
@@ -31,6 +32,10 @@ angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
     blockUIConfig.cssClass = 'block-ui block-ui-anim-fade consent-dialog';
     blockUIConfig.autoBlock = false;
   })
+  .config(['localStorageServiceProvider', function (localStorageServiceProvider) {
+    localStorageServiceProvider.setPrefix('jh');
```

Comment:
janusHangouts. That's the namespace of the application (it was named that way in the beginning).